corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -72,6 +91,15 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
                               placeholder: '{% trans "Click here to add a description" %}',
                             "></textarea>
           </td>
+          {% if fhir_integration_enabled %}
+          <td>
+            <textarea class="form-control vertical-resize" data-bind="
```

Comment:
Could I argue in favor of a text input instead of a textarea? I'm hoping that the JSON path won't be super long.

Reply:
absolutely, i thought these would be long. Updated.